### PR TITLE
ACS-4759 Replace default password hashing algorithm.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

File: repository/src/test/java/org/alfresco/repo/security/authentication/AuthenticationTest.java [hunk 1]
```diff
@@ -2076,44 +2074,49 @@ public void testRehashedPasswordOnAuthentication() throws Exception
 
     /**
      * For on premise the default is MD4, for cloud BCRYPT10
-     *
-     * @throws Exception
      */
-    public void testDefaultEncodingIsMD4() throws Exception
+    public void testDefaultEncodingIsMD4()
```
Comment (mpichura) [inline repository/src/test/java/org/alfresco/repo/security/authentication/AuthenticationTest.java:2078]: Shouldn't the method name change too?

Comment (krdabrowski) [inline repository/src/test/java/org/alfresco/repo/security/authentication/AuthenticationTest.java:2078]: As "MD4" is not the default encoding anymore, maybe method name should be fixed?

**Review by krdabrowski — APPROVED** [review]
LGTM

**Review by mpichura — APPROVED** [review]
Generally ok- had a small remark on method naming.

**Review by azakrzewski-hy — APPROVED** [review]

**Pull request merged**